app/models/concerns/court_cell_validations.rb

```diff
@@ -3,14 +3,6 @@
 module CourtCellValidations
   extend ActiveSupport::Concern
 
-  def court_cell_number=(court_cell_number)
-    details['court_cell_number'] = court_cell_number
-  end
-
-  def court_cell_number
-    details['court_cell_number']
-  end
-
```

Comment:
Should the concern also specify the `details_attributes` or do you do this for each case where the concern is included, and then call `details_attributes` just once per class? 🤔

Reply:
I think it makes sense for them to be decoupled like this, yeah. Even if there is a dependency there 🤔

Reply:
The comment is mostly me rambling whilst trying to understand how this works - it's all good 👍

Reply:
Cool beans 🏖️